<launch>
    <node pkg="rviz" type="rviz" name="rviz" args= "-d $(find sobit_follower)/config/rviz/follow_simulator.rviz"/>
    
    <node pkg="sobit_follower" type="sf_robot_pose_broadcaster" name="robot_pose_broadcaster" launch-prefix="xterm -e"/>
    <!-- follow_simulator -->
    <group ns = "follow_simulator">
        <node pkg="sobit_follower" type="sf_follow_simulator" name="follow_simulator" launch-prefix="xterm -e">
            <remap from="/follow_simulator/follow_position" to="/follow_position" />
            <rosparam>
                obstacle_size : 50
            </rosparam>
        </node>
    </group>
    <!-- path_generator -->
    <include file="$(find sobit_follower)/launch/follow_me/path_generator.launch"/>

    <node name="vel_plot_lin" pkg="rqt_plot" type="rqt_plot" args="/cmd_vel_mux/input/teleop/linear/x /cmd_vel_mux/input/teleop/angular/z" />
</launch>